### PR TITLE
📝 Document `getInternalContext` API

### PR DESCRIPTION
**Pull request by liywjl**
Add documentation on how to access internal context in `DD_LOG` and `DD_RUM`, and how it can be used to expose the `session_id` (and other attributes in `DD_LOG`)

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: packages/logs/README.md [hunk 1]
```diff
@@ -667,6 +667,33 @@ window.DD_LOGS && DD_LOGS.logger.setHandler(['<HANDLER1>', '<HANDLER2>'])
...
+### Access internal context
+
+After the Datadog browser logs SDK is initialized, you can access the internal context of the SDK. This will allow you to access the `session_id`.
+
+```
+getInternalContext (startTime?: 'number' | undefined)
+```
+You can optionally use `startTime` parameter to get the context of a specific time. If not return the current context is parameter is omitted.
+
+#### CDN async
```
Comment (bcaudan) [inline packages/logs/README.md:679]: **💭 thought:** ‏to be consistent with other sections, we should also have a paragraph for npm setup

File: packages/logs/README.md [hunk 2]
```diff
@@ -667,6 +667,33 @@ window.DD_LOGS && DD_LOGS.logger.setHandler(['<HANDLER1>', '<HANDLER2>'])
 
 **Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the SDK.
 
+### Access internal context
+
+After the Datadog browser logs SDK is initialized, you can access the internal context of the SDK. This will allow you to access the `session_id`.
```
Comment (bcaudan) [inline packages/logs/README.md:672]: **💬 suggestion:** ‏avoid temporal words, cf https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md
```suggestion
After the Datadog browser logs SDK is initialized, you can access the internal context of the SDK. This allows you to access the `session_id`.
```

File: packages/logs/README.md [hunk 3]
```diff
@@ -667,6 +667,33 @@ window.DD_LOGS && DD_LOGS.logger.setHandler(['<HANDLER1>', '<HANDLER2>'])
...
+
+```
+getInternalContext (startTime?: 'number' | undefined)
+```
+You can optionally use `startTime` parameter to get the context of a specific time. If not return the current context is parameter is omitted.
```
Comment (bcaudan) [inline packages/logs/README.md:677]: **💬 suggestion:** ‏phrasing
```suggestion
You can optionally use `startTime` parameter to get the context of a specific time. If the parameter is omitted, the current context is returned.
```

**Review by cswatt — APPROVED** [review]

**Review by BenoitZugmeyer — COMMENTED** [review]
Looks good, but:

* should we encourage people to use `getInternalContext`? I think we didn't document the `getInternalContext` to keep its usage... internal.
  * A `getSessionId()` would be more explicit and more future proof.

* if we chose to publish this documentation nonetheless, we should do the same in the RUM docs

File: packages/rum/README.md [hunk 4]
```diff
@@ -342,6 +342,44 @@ A service is an independent, deployable code repository that maps to a set of pa
 - If your browser application was constructed as a monolith, your RUM application has one service name for the application.
 - If your browser application was constructed as separate repositories for multiple pages, edit the default service names throughout the lifecycle of your application.
 
+### Access internal context
+
+After the Datadog browser logs SDK is initialized, you can access the internal context of the SDK. This allows you to access `session_id`, `application_id`, `user_action` and `view`.
```
Comment (bcaudan) [inline packages/rum/README.md:347]:
```suggestion
After the Datadog browser RUM SDK is initialized, you can access the internal context of the SDK. This allows you to access `session_id`, `application_id`, `user_action` and `view`.
```

Comment (bcaudan) [inline packages/rum/README.md:347]: **💬 suggestion:** `user_action` and `view`‏ does not seem very explicit to me, what about describing the exact structure of the internal context?

File: packages/rum/README.md [hunk 5]
```diff
@@ -342,6 +342,68 @@ A service is an independent, deployable code repository that maps to a set of pa
...
+
+```
+{
+  application_id : "xxx",
+  session_id : "xxx",
+  user_action: { id: "xxx" },
+  view : {
+    id : "xxx",
+    referrer : "",
+    url: "http://localhost:8080/"
```
Comment (bcaudan) [inline packages/rum/README.md:368]: **💬 suggestion:** ‏
```suggestion
    url: "http://localhost:8080/",
    name: "homepage"
```

**Review by bcaudan — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

File: packages/logs/README.md [hunk 6]
```diff
@@ -667,6 +667,44 @@ window.DD_LOGS && DD_LOGS.logger.setHandler(['<HANDLER1>', '<HANDLER2>'])
...
+  DD_LOGS.getInternalContext() // { session_id: "xxxx-xxxx-xxxx-xxxx" }
+})
+```
+
+##### CDN sync
+
+For CDN sync, use:
+
+```javascript
+DD_LOGS.getInternalContext() // { session_id: "xxxx-xxxx-xxxx-xxxx" }
```
Comment (BenoitZugmeyer) [inline packages/logs/README.md:705]: **🔨 warning:** ‏CDN sync should be `window.DD_LOGS && window.DD_LOGS.toto` (same for RUM)

Comment (liywjl) [reply packages/logs/README.md:705]:
```suggestion
window.DD_LOGS && window.DD_LOGS.getInternalContext() // { session_id: "xxxx-xxxx-xxxx-xxxx" }
```

File: packages/rum/README.md [hunk 7]
```diff
@@ -342,6 +342,69 @@ A service is an independent, deployable code repository that maps to a set of pa
...
+  DD_RUM.getInternalContext() // { session_id: "xxxx", application_id: "xxxx" ... }
+})
+```
+
+##### CDN sync
+
+For CDN sync, use:
+
+```javascript
+DD_RUM.getInternalContext() // { session_id: "xxxx", application_id: "xxxx" ... }
```
Comment (liywjl) [inline packages/rum/README.md:405]:
```suggestion
window.DD_RUM && window.DD_RUM.getInternalContext() // { session_id: "xxxx", application_id: "xxxx" ... }
```

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**